test/push-shared-code-with-vars/in/.keboola/manifest.json

```diff
@@ -0,0 +1,63 @@
+{
+  "version": 1,
+  "project": {
+    "id": %%TEST_KBC_PROJECT_ID%%,
+    "apiHost": "%%TEST_KBC_STORAGE_API_HOST%%"
+  },
+  "sortBy": "path",
+  "naming": {
+    "branch": "{branch_name}",
+    "config": "{component_type}/{component_id}/{config_name}",
+    "configRow": "rows/{config_row_name}",
+    "sharedCodeConfig": "_shared/{target_component_id}",
+    "sharedCodeConfigRow": "codes/{config_row_name}",
+    "variablesConfig": "variables",
+    "variablesValuesRow": "values/{config_row_name}"
+  },
+  "allowedBranches": [
+    "__all__"
+  ],
+  "ignoredComponents": [],
+  "branches": [
+    {
+      "id": %%TEST_BRANCH_MAIN_ID%%,
+      "path": "my-branch"
+    }
+  ],
+  "configurations": [
+    {
+      "branchId": %%TEST_BRANCH_MAIN_ID%%,
+      "componentId": "keboola.shared-code",
+      "id": "%%TEST_NEW_TICKET_1%%",
```

Comment:
Ak chcem v teste nove ID, tak mozem pouzit `TEST_NEW_TICKET_x` ... pri prvom vyskyte sa vygeneruje nove ID pomocou API, .... pri dalsich vyskytoch sa iba nahradi (to tu uz bolo, len vysvetlenie).